lbry/lbry/conf.py

```diff
@@ -630,9 +630,13 @@ def log_file_path(self):
 
 
 def get_windows_directories() -> typing.Tuple[str, str, str]:
-    from lbry.winpaths import get_path, FOLDERID, UserHandle  # pylint: disable=import-outside-toplevel
+    from lbry.winpaths import get_path, FOLDERID, UserHandle, \
+        PathNotFoundException  # pylint: disable=import-outside-toplevel
 
-    download_dir = get_path(FOLDERID.Downloads, UserHandle.current)
+    try:
+        download_dir = get_path(FOLDERID.Downloads, UserHandle.current)
+    except PathNotFoundException:
+        download_dir = os.getcwd()
```

Comment:
This seems like a substantial change, perhaps it should be its own PR so we can get it properly documented in the release? (bonus: we can release 0.48.1 right away with that PR to test that gitlab releases work)

Reply:
This change is not that substantial. This code is only called in one place, in `cli.py` on line 290:
```
conf = Config.create_from_arguments(args)
```
This is five lines into `main()`. Previous to this change, the `PathNotFoundException` that's raised would bubble all the way up and immediately crash the sdk. So the only people affected by this change are people for whom the sdk does not work at the moment.

I'm happy to move this into a separate PR, but this code is here because the binary does not run on our windows CI runner without it (probably because the runner is a server and does not have a Downloads folder set). Do you really want me to separate out this code just so we can merge it ahead of the rest of this PR?

Reply:
The changed behavior is that before it used to crashed with some clue of why it crashed (can't find download directory) and now it will silently work and perhaps place files where the user doesn't want / won't know to look for.

Maybe it's fine to hide this change in this PR but in general fixing something from previously crashing to silently working (but doing potentially unexpected things) I would consider a substantial change. We've gotten beef from app team and @tzarebczan for lesser changes :-D

Reply:
You're right. I'll let Tom know about this